modules/geolocation/geolocation.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+class Geolocation:
+    """Locates the geographical position of a set of pixels"""
+
+    # TODO Class members
+    def __init__(self):
+
+        self.__cameraOrigin3o = np.array([0.0, 0.0, 2.0])
+        self.__cameraDirection3c = np.array([0.0, 0.0, -1.0])
+        self.__cameraOrientation3u = np.array([1.0, 0.0, 0.0])
+        self.__cameraOrientation3v = 1 * np.cross(self.__cameraDirection3c, self.__cameraOrientation3u)
+        self.__cameraResolution = np.array([1000, 1000])  # TODO Make global?
+        self.__referencePixels = np.array([[0, 0],
+                                           [0, 1000],
+                                           [1000, 0],
+                                           [1000, 1000]])
+
+        self.__pixelsToGeographicals = np.array([[[0, 0], [0, 0]],
+                                                 [[1, 1], [1, 1]],
+                                                 [[2, 2], [2, 2]],
+                                                 [[3, 3], [3, 3]]])
+
+        return
+
+    # TODO Placeholder, add functionality once we figure out how to convert raw plane data
+    def convert_input(self):
```

Comment:
Does this function call all functions required to convert camera points to geopoints? If not, then we need a function like that which is called by the parent after instantiation. Like the builder pattern, if you're familiar with that.

Reply:
This function converts raw plane data into something the module can do work on (see [documentation](https://docs.google.com/document/d/1VKYIrmWJYfpLPCAjQPiH1J5fnQPtNcjEC2NE5Eg4QxU/edit?usp=sharing)).

Reply:
I haven't yet thought of the name of the function that will run the entire thing from start to finish.